src/app/containers/FrontPageMain/index.jsx

```diff
@@ -25,28 +25,30 @@ const itemColumns = {
   group1: 6,
   group2: 6,
   group3: 6,
```

Comment:
Since `itemColumns` is now identical to `mainColumns`, we could consider removing the Grid item